CODEOWNERS

```diff
@@ -1,27 +1,37 @@
 *                           @gakonst
 bin/                        @onbjerg
-crates/blockchain-tree      @rakita @rkrasiuk
+crates/blockchain-tree/     @rakita @rkrasiuk @mattsse @Rjected
+crates/blockchain-tree-api/ @rakita @rkrasiuk @mattsse @Rjected
 crates/cli/                 @onbjerg @mattsse
-crates/consensus            @rkrasiuk @mattsse @Rjected
-crates/exex                 @onbjerg @shekhirin
-crates/metrics              @onbjerg
+crates/config/              @onbjerg
+crates/consensus/           @rkrasiuk @mattsse @Rjected
+crates/e2e-test-utils/      @mattsse @Rjected
+crates/engine-primitives/   @rkrasiuk @mattsse @Rjected
+crates/errors/              @mattsse
+crates/ethereum/            @mattsse @Rjected
+crates/ethereum-forks/      @mattsse @Rjected
+crates/etl/                 @joshieDo @shekhirin
+crates/evm/                 @rakita @mattsse @Rjected @DaniPopes
+crates/exex/                @onbjerg @shekhirin
+crates/fs-util/
+crates/metrics/             @onbjerg
 crates/net/                 @emhane @mattsse @Rjected
 crates/net/downloaders/     @onbjerg @rkrasiuk
 crates/node/                @mattsse @Rjected @onbjerg
 crates/node-core/           @mattsse @Rjected @onbjerg
-crates/node-ethereum/       @mattsse @Rjected
+crates/optimism/            @mattsse @Rjected @fgimenez
 crates/payload/             @mattsse @Rjected
-crates/prune                @shekhirin @joshieDo
+crates/primitives/          @DaniPopes @Rjected
+crates/prune/               @shekhirin @joshieDo
 crates/revm/                @mattsse @rakita
 crates/rpc/                 @mattsse @Rjected
 crates/stages/              @onbjerg @rkrasiuk @shekhirin
-crates/stages-api/          @onbjerg @rkrasiuk @shekhirin
-crates/static-file          @joshieDo @shekhirin
+crates/static-file/         @joshieDo @shekhirin
+crates/static-file-types/   @joshieDo @shekhirin
 crates/storage/             @rakita @joshieDo @shekhirin
```

Comment:
think we need to unpack this folder at some point, it contains so much code by diff authors i think it makes sense to be granular

Reply:
any suggestions on owners for each subfolder?

Reply:
the owner set will be roughly the same, but i think it makes sense to be granular still, esp when/if these are refactored, so i think

```codeowners
crates/storage/codecs              @joshieDo
crates/storage/db                      @joshieDo @rakita 
crates/storage/db-api               @joshieDo @rakita # think these were written primarily by joshie/dragan?
crates/storage/libmdbx-rs        @rakita # think dragan has primarily been maintaining this
crates/storage/nippy-jar           @joshieDo @shekhirin 
crates/storage/provider            @rakita @joshieDo @shekhirin 
```

storage-api/errors/db-common i have no idea

Reply:
no idea why formatting is so bad, they are aligned when editing the comment

Reply:
copied this, tried to fill all the subfolders for storage